x/ccv/child/module.go

```diff
@@ -249,6 +278,32 @@ func (am AppModule) OnChanOpenInit(
 		return err
 	}
 
+	///////////////////////////////////////////////////
+	// Initialize distribution token transfer channel
+	// NOTE the handshake for this channel is handled by
+	//      the ibc-go/transfer module
+
+	transferPortID := "transfer"
+	am.keeper.SetDistributionTransmissionPort(ctx, transferPortID)
+
+	distChan := channeltypes.Channel{
+		State:          channeltypes.UNINITIALIZED,
+		Ordering:       channeltypes.UNORDERED,
+		Counterparty:   counterparty,          // XXX is it correct to reuse
+		ConnectionHops: connectionHops,        // XXX is it correct to reused
+		Version:        transfertypes.Version, // XXX verify that this is correct
```

Comment:
@AdityaSripal have I set `Counterparty`, `ConnectionHops`, and `Version` correctly here?

Reply:
ConnectionHops and Version is correct, Counterparty is wrong. see above comment